.github/workflows/build.yaml

```diff
@@ -0,0 +1,197 @@
+      uses: actions/checkout@v2
+    - name: "Get SHA"
+      id: get_sha
+      run: echo "::set-output name=current_sha_short::$(git rev-parse --short HEAD)"
+    - name: "Download htmltest cache artifact"
+      uses: actions/download-artifact@v2
+      with:
+        name: hugo-cache
+        path: ${{ env.CACHE_LOCATION }}
+    - name: "Push new htmltest cache cache"
+      id: htmltest-cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ env.CACHE_LOCATION }}
+        key: htmltest-cache-${{ steps.get_sha.outputs.current_sha_short }}
+
+  hugo-test:
+    name: "Test Hugo Site"
+    runs-on: ubuntu-latest
+    needs: htmltest-cache-update
+    steps:
+    - name: "Checkout source code at current commit"
+      uses: actions/checkout@v2
+    - name: "Get SHA"
+      id: get_sha
+      shell: bash
+      run: echo "::set-output name=current_sha_short::$(git rev-parse --short HEAD)"
+    - name: "Download Hugo site artifact"
+      uses: actions/download-artifact@v2
+      with:
+        name: hugo-site
+        path: ./public
+    - name: "Extract cache pull location"
+      id: get_cache_pull_location
+      run: |
+        export cache_pull_location=$( "${{ env.CACHE_LOCATION }}" | sed 's|\(.*\)/.*|\1|')
+        echo "::set-output name=cache-pull-location::cache_pull_location"
+    - name: "Pull new htmltest cache"
+      id: htmltest-cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.get_cache_pull_location.outputs.cache-pull-location }}
+        key: htmltest-cache-${{ steps.get_sha.outputs.current_sha_short }}
+    - name: "Download Docker image artifact"
+      uses: actions/download-artifact@v2
+      with:
+        name: docker-image
+        path: /tmp/
+    - name: "Import Image"
+      run: docker load --input /tmp/image.tar
+    - name: "Hugo Site Smoketest, No Training Wheels"
+      run: make real-clean smoketest
+
+  hugo-deploy:
+    name: "Deploy New Site Resources"
+    runs-on: ubuntu-latest
+    needs: hugo-test
+    if: github.event_name == 'release' && github.event.action == 'published'
+    steps:
+    - name: "Checkout source code at current commit"
+      uses: actions/checkout@v2
+    - name: "Get SHA"
+      id: get_sha
+      shell: bash
+      run: echo "::set-output name=current_sha_short::$(git rev-parse --short HEAD)"
+    - name: "Download Docker image artifact"
+      uses: actions/download-artifact@v2
+      with:
+        name: docker-image
+        path: /tmp/
+    - name: "Import Image"
+      run: docker load --input /tmp/image.tar
+    - name: "Download Hugo site artifact"
+      uses: actions/download-artifact@v2
+      with:
+        name: hugo-site
+        path: ./public
+    - name: "Pull new htmltest cache"
+      id: htmltest-cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ env.CACHE_LOCATION }}
+        key: htmltest-cache-${{ steps.get_sha.outputs.current_sha_short }}
+    - name: "Push Site to S3"
+      run: |
+        set +e -x
```

Comment:
Instead set the defaults

See https://github.com/cloudposse/actions/blob/master/.github/workflows/test-command.yml#L6-L9